packages/debugger/src/panels/breakpoints/index.ts

```diff
@@ -54,12 +55,16 @@ export class Breakpoints extends Panel {
       })
     );
 
-    this.addWidget(header);
+    //this.addWidget(header);
     this.addWidget(body);
-
     this.addClass('jp-DebuggerBreakpoints');
   }
 
+  get header(): PanelHeader {
```

Comment:
Let's add a docstring for consistency?

Reply:
I'm working on it, this PR needs 1-2 more commits to be ready for review.